project-template/index.template.js

```diff
@@ -1 +1,17 @@
-console.log('I am the bot.');
+const { registerModule } = require('nay');
+
```

Comment:
We import the `registerModule()` function from the `nay`, but the function is in the `core.js` file.
Don't we need to make anything about that inside the `core.js` file (export a function)?

Reply:
Nice